README.md

```diff
@@ -1,24 +1,73 @@
-* [Compare Editor's Copy to Working Group Draft](https://tlswg.github.io/tls-subcerts/#go.draft-ietf-tls-subcerts.diff)
 
-## Building the Draft
+# AUTH48 for RFC-to-be 9345 <draft-ietf-tls-subcerts-15>
+This repository is for managing document updates for RFC-to-be 9345 during AUTH48. 
 
-Formatted text and HTML versions of the draft can be built using `make`.
+Please see the [License](https://github.com/rfc-editor/draft-ietf-tls-subcerts/blob/master/LICENSE.md). The [IETF Note Well](https://github.com/rfc-editor/draft-ietf-tls-subcerts/blob/master/note-well.md) applies.
 
-```sh
-$ make
-```
+GitHub features that will be used:
+* Issue tracking
+* Pull requests
+* @mentions
 
-This requires that you have the necessary software installed.  See
-[the instructions](https://github.com/martinthomson/i-d-template/blob/master/doc/SETUP.md).
+Authors, AD, WG chairs, and document shepherd, please ensure that your GitHub notification settings are set such that you can participate promptly in discussions of issues and pull requests. For more information, please see [Configuring notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications).
 
+This document is being edited in Markdown.  Once the content is stable, the Markdown will be converted to XML and formatted as an RFC.  Your final approval means you approve both the content and format.
 
-## Contributing
+## Issue Tracking
+The initial version of rfc9345-to-be.md is a copy of draft-ietf-tls-subcerts.md that was approved for publication. 
+
+The current version of rfc9345-to-be.md has been edited minimally for correctness. See PR #NNN.
+
+The initial version of the file has been edited minimally for correctness.  The RFC Production Center's suggested updates have been added as PRs.  Open-ended questions and FYIs have been added as issues.  Please accept the PRs or propose alternate updates.  For the issues, provide your answers in the comments or add PRs to address the issues.  Use the issue labels to indicate their status:
+* rfced - initial label of RFC Editor questions and FYIs (keep this label if the issue is still active).
+* question - a follow-up question has been added to the issue discussion. Use @mention to identify the participant who should answer. 
+* AD approval required - the resolution of the issue requires the AD to review and approve the change.
+* rfced-ready - the RFC Editor can update the document and/or close the issue based on the issue discussion.
+
+Feel free to add new issues and PRs for new edits and questions. 
+
+The editor will close the issue when it has been addressed either by accepting a pull request or simply closing the issue if no changes are needed. 
+
+## Pull Requests
+If you would like to submit changes, please create a pull request and identify one or more issues that the pull request closes in the comments. 
```

Comment:
This process seems backwards.  What I would expect is:

* RPC sends a PR with edits to the MD
* Editors make comments on / changes to the PR
* When the editors approve the changes, they signify this by merging the PR